src/transformers/models/auto/tokenization_auto.py

```diff
@@ -930,7 +930,7 @@ def from_pretrained(cls, pretrained_model_name_or_path, *inputs, **kwargs):
 
         model_type = config_class_to_model_type(type(config).__name__)
         if model_type is not None:
-            tokenizer_class_py, tokenizer_class_fast = TOKENIZER_MAPPING[type(config)]
+            tokenizer_class_py, tokenizer_class_fast = TOKENIZER_MAPPING.get(type(config), PreTrainedTokenizerFast)
```

Comment:
💯

Reply:
Also TOKENIZER_MAPPING can return a tuple with (SomeTokenizer, None) and so PreTrainedTokenizerFast would not be set here